exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -172,22 +214,27 @@ func (prwe *prwExporter) PushMetrics(ctx context.Context, md pmetric.Metrics) er
 	case <-prwe.closeChan:
 		return errors.New("shutdown has been called")
 	default:
-
-		tsMap, err := prometheusremotewrite.FromMetrics(md, prwe.exporterSettings)
-		if err != nil {
-			prwe.telemetry.recordTranslationFailure(ctx)
-			prwe.settings.Logger.Debug("failed to translate metrics, exporting remaining metrics", zap.Error(err), zap.Int("translated", len(tsMap)))
+		if !prwe.enableSendingRW2 {
+			return prwe.pushMetricsV1(ctx, md)
 		}
+		switch prwe.RemoteWriteProtoMsg {
+		case RemoteWriteProtoMsgV1:
+			return prwe.pushMetricsV1(ctx, md)
+		case RemoteWriteProtoMsgV2:
+			// RW2 case
+			tsMap, symbolsTable, err := prometheusremotewrite.FromMetricsV2(md, prwe.exporterSettings)
```

Comment:
Why not make a corresponding `prwe.pushMetricsV2` method for symmetry?

Reply:
Done.